recipe/build.sh

```diff
@@ -2,12 +2,9 @@
 set -ex
 
 pushd HiggsAnalysis/CombinedLimit
-make CONDA=1 -j${CPU_COUNT}
-cp -rv build/bin ${PREFIX}
-cp -v build/lib/*${SHLIB_EXT} ${PREFIX}/lib
-cp -v build/lib/*.pcm ${PREFIX}/lib
-cp -v build/lib/*.rootmap ${PREFIX}/lib
-mkdir -p ${PREFIX}/include/HiggsAnalysis/CombinedLimit
-cp -rv interface ${PREFIX}/include/HiggsAnalysis/CombinedLimit/
-cp -rv build/lib/python/HiggsAnalysis $(python3 -c "import sysconfig; print(sysconfig.get_path('platlib'))")/
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_PREFIX_PATH=$PREFIX ..
+make -j${CPU_COUNT}
+make install
```

Comment:
```suggestion
cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_PREFIX_PATH=$PREFIX ..
cmake -LH build
cmake --build build --clean-first --parallel="${CPU_COUNT}"
cmake --install build
```

You can simplify this a lot by just doing a standard out of source build with modern CMake. So in addition to this suggestion I'd also skip the `pushd` and `popd` (but I can't suggest edits here given the range GitHub gives).

So the entire `build.sh` file would be

```bash
#!/bin/bash
set -ex

cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_PREFIX_PATH=$PREFIX -S HiggsAnalysis/CombinedLimit -B build
cmake -LH build
cmake --build build --clean-first --parallel="${CPU_COUNT}"
cmake --install build
```

Reply:
I figured there was something more slick, thanks for providing it